kb/prometheus-grafana-setup-cloud.md

```diff
@@ -0,0 +1,252 @@
+---
+slug: prometheus-grafana-setup-cloud
+title: Set up Grafana with Temporal Cloud Observability to view metrics
```

Comment:
true; but Observability is what the label is on the Cloud UI so this is a way to start getting users to make these associations: Observability > metrics. Since the title does have metrics, this will show in the results. Happy to change it if you both feel strongly about it.

Reply:
I'm good with capitalizing the word when referring to its appearance in the UI (in fact, [you should](https://developers.google.com/style/ui-elements#formatting)). But when referring to the observability feature, I do recommend lowercase.